src/com/epam/javatrainings/classdecomposition/palmetto/model/Ingredients/Tomato.java

```diff
@@ -0,0 +1,7 @@
+package com.epam.javatrainings.classdecomposition.palmetto.model.Ingredients;
+
+public class Tomato extends Ingredient{
```

Comment:
Ingredient abstraction is overused, you can just create an enum or just leave one Ingredient class
Tomato or others are instances of Ingredient not types.